Flaky reports test

When running `test_period_totals_breakdown_report` we occasionally get a different order for the assertion below. 


```
E       AssertionError: assert [['SQUARE_ONL...rePOS', 1100]] == [['SquarePOS'...ONLINE', 580]]
E         At index 0 diff: ['SQUARE_ONLINE', 580] != ['SquarePOS', 1100]
```

--- a/uobtheatre/reports/reports.py
+++ b/uobtheatre/reports/reports.py
@@ -127,6 +127,9 @@ class PeriodTotalsBreakdown(Report):
             )
             row[1] += payment.value
 
+        # Sort alphabetically
+        provider_totals_set.data.sort(key=lambda provider: provider[0])
+
         self.datasets.extend(
             [
                 provider_totals_set,

--- a/uobtheatre/reports/test/test_reports.py
+++ b/uobtheatre/reports/test/test_reports.py
@@ -118,7 +118,7 @@ def create_fixtures():
     payment_1 = PaymentFactory(
         pay_object=booking_1,
         value=booking_1.total,
-        provider=payment_methods.SquarePOS.__name__,
+        provider=payment_methods.SquarePOS.name,
         provider_payment_id="square_id",
         app_fee=booking_1.misc_costs_value(),
         provider_fee=10,
@@ -216,8 +216,8 @@ def test_period_totals_breakdown_report():
     assert report.datasets[0].name == "Provider Totals"
     assert len(report.datasets[0].headings) == 2
     assert report.datasets[0].data == [
-        ["SquarePOS", 1100],
         ["SQUARE_ONLINE", 580],
+        ["SQUARE_POS", 1100],
     ]
 
     assert report.datasets[1].name == "Production Totals"
@@ -243,7 +243,7 @@ def test_period_totals_breakdown_report():
             "2021-09-08 00:00:01",
             str(booking_1.id),
             "1100",
-            "SquarePOS",
+            "SQUARE_POS",
             "square_id",
         ],
         [

